internal/connect/main.go

```diff
@@ -17,10 +17,13 @@ var (
 	CFG Config
 	// Debug logger for debugging output
 	Debug *log.Logger
+	// QuietOut is used to simplify --quiet option
+	QuietOut *log.Logger
```

Comment:
hmmm... sounds good.

Reply:
no.
wait.
I added it here again to get the "Executing ..." output from zypper calls automatically if not quiet.
if we reverse the logic and name this logger "Info" this would mean that it's enabled by default unless `--quiet` is used.
this results in "Executing ..." being printed in `SUSEConnect` case for all zypper calls.
I think `QuietOut` is still a bit more clear if you consider above.

Reply:
then the name `Info` can't be used as a direct replacement for `QuietOut`. maybe we'll find some better alternative later.

Reply:
If the name is changed to `Info`, the logger will still be disabled when `SUSEConnect` runs with the current logic. Anyway it's not so important - just `QuietOut` is a peculiar name.

Reply:
It would but I think it would be confusing. By default, most tools output "info" level messages without any additional options; so if we name it `Info` and make it not print anything by default it would be strange (at least to me).

Reply:
okay thanks. I understand now.